flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaDynamicTableFactoryTest.java

```diff
@@ -851,6 +851,17 @@ public void testPrimaryKeyValidation() {
             assertEquals(error, t.getCause().getMessage());
         }
 
+        try {
+            createTableSink(pkSchema, getKeyValueOptions());
```

Comment:
I think you can already use something along the lines of

```java
assertThatThrownBy(() -> createTableSink(pkSchema, getKeyValueOptions())
    .isInstanceOf(...)
    .hasMessageContaining()
```

Reply:
I don't think this is resolved you are still using the `junit` construct and not `assertj` ;)

Reply:
Maybe you have to rebase to the latest master to pull the change introducing assertj